manon/components/footer-two-thirds-one-third.scss

```diff
@@ -0,0 +1,76 @@
+@use "../mixin/apply-from-theme";
```

Comment:
```suggestion
@use "../mixins/apply-from-theme";
```

Reply:
Renamed the file mixin to mixins in a previous pr. Therefor we need to change the reference in this file as well.